cmd/aws-application-networking-k8s/main.go

```diff
@@ -151,20 +151,14 @@ func main() {
 		setupLog.Fatalf("route controller setup failed: %s", err)
 	}
 
-	serviceImportReconciler := controllers.NewServceImportReconciler(mgr.GetClient(), mgr.GetScheme(),
-		mgr.GetEventRecorderFor("ServiceImport"), finalizerManager, latticeDataStore)
-
-	if err = serviceImportReconciler.SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "ServiceImport")
-		os.Exit(1)
+	err = controllers.RegisterServiceImportReconciler(ctrlLog.Named("serviceimport"), mgr, latticeDataStore, finalizerManager)
+	if err != nil {
```

Comment:
Should we unify the name  `Reconciler` and `Controller` to only use name `Controller`? (Maybe in a separate PR)

Reply:
This is my bad, will fix in next commit!